internal/cmd/beta/beta_cmd.go

```diff
@@ -54,6 +70,8 @@ All commands use the runme.yaml configuration file.`,
 	pFlags := cmd.PersistentFlags()
 	pFlags.StringVar(&cFlags.filename, "filename", "", "Name of the Markdown file to run blocks from.")
 	pFlags.StringSliceVar(&cFlags.categories, "category", nil, "Run blocks only from listed categories.")
+	pFlags.BoolVar(&cFlags.silent, "silent", false, "Silent mode. Do not error messages.")
+	pFlags.BoolVar(&cFlags.subtle, "subtle", false, "Explicitly allow delicate operations to prevent misuse")
```

Comment:
Nit: it's first time I see "subtle" in such context. Typically it's more like `--unsafe`.

Reply:
I was between subtle and insecure. I think I'll use the latter since we already made use in non-beta cmds.